tests/test_mock_context_standalone.py

```diff
@@ -63,6 +63,21 @@ def test_attr_scalar(context_scalar_return):
     assert context_scalar_return.t == 5
 
 
+@pytest.mark.parametrize('inp', [None, [], [[]]])
+def test_context_empty_input(inp):
+    meta_data = MockMetaData(
+        script_code_wrapper_function=udf_wrapper,
+        input_type='SCALAR',
+        input_columns=[],
+        output_type='RETURNS',
+        output_columns=[Column('t', int, 'INTEGER')]
+    )
+    _ = StandaloneMockContext(inp, meta_data)
+    # There is nothing to test here apart from successful creation of the
+    # context object. This internally has some checks that need to be passed.
+    pass
```

Comment:
Maybe the test could assert that the constructor does not raise an exception? 
https://stackoverflow.com/questions/4319825/python-unittest-opposite-of-assertraises

Reply:
I don't understand why it is so important to make a distinction between a failure and an error. Either way, pytest will point out where the problem is. If you convert an exception to an assert failure you are more likely to hide the actual cause of the error.